lightning-block-sync/src/poll.rs

```diff
@@ -136,8 +136,11 @@ impl ValidatedBlockHeader {
 
 		if let Network::Bitcoin = network {
 			if self.height % 2016 == 0 {
-				let previous_work = previous_header.header.work();
-				if work > (previous_work << 2) || work < (previous_work >> 2) {
+				let target = self.header.target();
+				let previous_target = previous_header.header.target();
+				let min_target = previous_target >> 2;
+				let max_target = previous_target << 2;
```

Comment:
I guess then we can start using  `(min/max)_difficulty_transition_threshold` after updating.